uchiwa/config/config.go

```diff
@@ -160,6 +167,13 @@ func (c *Config) initUchiwa() {
 		c.Uchiwa.Users = append(c.Uchiwa.Users, auth.User{Username: c.Uchiwa.User, Password: c.Uchiwa.Pass, FullName: c.Uchiwa.User})
 	}
 
+	// audit
+	if c.Uchiwa.Audit.Level != "verbose" && c.Uchiwa.Audit.Level != "disabled" {
+		c.Uchiwa.Audit.Level = "default"
+	}
+	if c.Uchiwa.Audit.Logfile == "" {
+		c.Uchiwa.Audit.Logfile = "/var/log/sensu-enterprise/sensu-enterprise-dashboard-audit.log"
```

Comment:
Let's update this logfile location to `/var/log/sensu/sensu-enterprise-dashboard-audit.log`